float64 sigma_h
float64 sigma_s
float64 sigma_v
---
float64 h
float64 s
float64 v
